executor/ddl.go

```diff
@@ -486,7 +485,7 @@ func (e *DDLExec) getRecoverTableByTableName(tableName *ast.TableName) (*model.J
 		// Check GC safe point for getting snapshot infoSchema.
 		err = gcutil.ValidateSnapshotWithGCSafePoint(job.StartTS, gcSafePoint)
 		if err != nil {
-			return nil, nil, err
+			return nil, nil, errors.Errorf("Can't find drop/truncate table '%s' in GC safe point %s", tableName.Name.O, model.TSConvert2Time(gcSafePoint).String())
```

Comment:
```suggestion
			return nil, nil, errors.Errorf("Can't flashback table '%s', it has been Garbage Collected", tableName.Name.O)
```

Reply:
If the user flashback a table that even not exist before, the message will be wired?